ext/configuration.h

```diff
@@ -86,7 +86,7 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(BOOL, DD_TRACE_MEASURE_COMPILE_TIME, "true")                                                        \
     CONFIG(BOOL, DD_TRACE_DEBUG, "false")                                                                      \
     CONFIG(BOOL, DD_TRACE_ENABLED, "true", .ini_change = ddtrace_alter_dd_trace_disabled_config)               \
-    CONFIG(BOOL, DD_TRACE_TELEMETRY_ENABLED, "false", .ini_change = zai_config_system_ini_change)              \
+    CONFIG(BOOL, DD_TRACE_TELEMETRY_ENABLED, "true", .ini_change = zai_config_system_ini_change)               \
```

Comment:
I just realize that now but all languages (but JS) use `DD_TRACE_TELEMETRY_ENABLED` . We should change

Reply:
Done :-)